### ISSUE TITLE
fails to build on ubuntu 22.04 and debian 11 (ncurses failure)

### ISSUE THREAD

**Issue by cleary** [yaxu/feedforward#37]
I'm currently getting the following issue on debian 11 and ubuntu 22.04:
```
"Configuring ncurses-0.2.16...", "Preprocessing library for ncurses-0.2.16..", "c2hs: C header contains errors:", "", "dist/build/UI/NCurses/Enums.chs.h:140: (column 25) [ERROR]  >>> Unknown identifier!", "  Cannot find a definition for `KEY_EVENT' in the header file."
```

This issue made an appearance in this debian bug from Sept 2020:
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=971187

Which wasn't resolved and subsequently resulted in the `haskell-ncurses` package being removed from debian altogether

**Comment by yaxu** [comment]
Yes this library is not maintained unfortunately, and developing/supporting feedforward isn't high on my priorities at the moment.

If you download the tarball from https://hackage.haskell.org/package/ncurses

and then delete the offending line, it compiles and works fine.

**Comment by ndr-brt** [comment]
Related to #22

**Comment by yaxu** [comment]
Feedforward has been moved to the tidal repo:
https://github.com/tidalcycles/Tidal/tree/main/feedforward
It's now moved to hscurses instead of ncurses
